common/djangoapps/entitlements/rest_api/v1/views.py

```diff
@@ -580,11 +575,6 @@ def _process_revoke_and_downgrade_to_audit(self, course_entitlements, user_id, r
                                                                          entitled_course_ids,
                                                                          username))
             return
-        log.info('B2C_SUBSCRIPTIONS: Starting revoke_entitlements_and_downgrade_courses_to_audit for user [%s] and '
```

Comment:
Removed this log from both the view and task file because I have added a log in the `revoke_entitlements_and_downgrade_courses_to_audit` task when the task is received, hence these logs were redundant.